samtools.wdl

```diff
@@ -23,29 +24,38 @@ task BgzipAndIndex {
     }
 
     runtime {
-        docker: "quay.io/biocontainers/tabix:" + dockerTag
+       docker: dockerImage
     }
 }
 
 task Index {
     input {
         File bamFile
-        String bamIndexPath
-
-        String dockerTag = "1.8--h46bd0b3_5"
+        String? outputBamPath
```

Comment:
Since this input is optional, the command will not work. Eg. if I don't specify this `mkdir -p $(dirname )` will get run.
```suggestion
        String outputBamPath = basename(bamFile)
```